bindings_node/src/conversations.rs

```diff
@@ -258,7 +258,7 @@ impl Conversations {
 
     let num_groups_synced = self
       .inner_client
-      .sync_all_welcomes_and_groups(&conn)
+      .sync_all_welcomes_and_groups(&conn, None)
```

Comment:
I think a follow up PR needs to be done not only bring the consent filtering to the sync but also to the list functions on the node and browser bindings